nautobot_floor_plan/migrations/0008_add_axis_step.py

```diff
@@ -2,7 +2,7 @@
 
 from django.db import migrations, models
 
-import nautobot_floor_plan.utils
+import nautobot_floor_plan.utils.utils
```

Comment:
nit: I'm not a big fan of `.utils.utils`. Maybe something like `.utils.general` or similar?